### ISSUE TITLE
is user_id field is same data as auditable_id in auditsTable?

### ISSUE THREAD

**Issue by johndavidft1** [owen-it/laravel-auditing#601]
| Q                 | A
| ----------------- | ---
| Bug?              |yes
| New Feature?      | no|yes
| Framework         | Laravel|Lumen
| Framework version | 7.22.4
| Package version   | 9.0
| PHP version       | 7.4.4

#### Actual Behaviour

Upon Registration in laravel app as a new User,  auditsTable gives me a record that **_user_id_** and **_user_type_** field is NULL..but the rest of the field is good.

**### Just wondering if **_user_id_** and **_auditable_id_** are same data in auditsTable??**

If that is indeed the case, I would like to represent **_auditable_id_** in my table.blade.php as user_name 

#### Expected Behaviour

user_type and user_id columns to populate when NEW user is registering my laravel application.

#### Steps to Reproduce

1. Install Laravel Auditing in a Laravel project.
2. setup the audit package
3. Implement Auditable on UserModel
4. try to register new user

#### Possible Solutions

**### Relating auditable_id in userTable**

![auditable_id links in userTable name field](https://user-images.githubusercontent.com/55346699/89531360-b24b7a00-d822-11ea-864a-e61a869a4a0b.jpg)

I have tried to link AuditModel **_auditable_id_** field to link it to UserModel/Table..
In my Audit.php(MODEL) I created a function named auditable_owner.

` public function auditable_owner()
     {
         return $this->belongsTo('App\User', 'auditable_id'); 
                                          
     }`

Now that audit_id can link to userTable, In my **_table.blade.php_** which is views, I tried calling the auditable_owner function and transform its corresponding id into a user name.

`<td>{{ $audits->auditable_owner['name'] }}</td>`

and it works. **Just want to know auditable_id is the id of the user...**

**Comment by parallels999** [comment]
>Just want to know if auditable_id field storing the id of the user...
>Just wondering if user_id and auditable_id are same data in auditsTable??

No, it can be the same if you are auditing the same user model that is on session login, but that would only be a coincidence of data
If you are getting the same yo have a implementation issue
